src/Symfony/Component/Security/Core/composer.json

```diff
@@ -19,6 +19,7 @@
         "php": ">=5.3.9",
         "symfony/polyfill-php55": "~1.0",
         "symfony/polyfill-php56": "~1.0",
+        "symfony/polyfill-php70": "~1.0",
```

Comment:
this one needs to be removed in 3.0 as its only there for the deprecated SecureRandom class